WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -90,24 +90,31 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 					trackedWallets.Remove(finishedCoinJoin.Wallet.WalletName);
 					finishedCoinJoin.CancellationTokenSource.Dispose();
 
-					var finishedCoinJoinTask = finishedCoinJoin.CoinJoinTask;
-					if (finishedCoinJoinTask.IsCompletedSuccessfully)
+					try
```

Comment:
Is it possible that a wallet is *closed* and *finished* at the same time? If so, then L80 and L91 may dispose `CancellationTokenSource` twice, I guess.

I don't understand the code good enough to verify that so feel free to ignore this if it can't happen.

Reply:
It could be, sure, but disposing twice is not a problem (I have verified it) because programmers have to implement the pattern to allow it to be called multiple times. But again, i have checked anyway and it doesn't fail.